app/controllers/account/billing-info/invoices/list.js

```diff
@@ -0,0 +1,156 @@
+import Controller from '@ember/controller';
+import EmberTableControllerMixin from 'open-event-frontend/mixins/ember-table-controller';
+
+export default class extends Controller.extend(EmberTableControllerMixin) {
+
+
+upcomingInvoiceColumns = [
+  {
+    name      : 'Invoice ID',
+    valuePath : 'invoiceId'
+  },
+  {
+    name      : 'Event Name',
+    valuePath : 'eventName'
+  },
+  {
+    name      : 'Date Issued',
+    valuePath : 'dateIssued'
+  },
+  {
+    name      : 'Outstanding Amount',
+    valuePath : 'amount'
+  },
+  {
+    name      : 'View Invoice',
+    valuePath : 'invoiceLink'
+  }
+];
+
+paidInvoiceColumns = [
+  {
+    name      : 'Invoice ID',
+    valuePath : 'invoiceId'
+  },
+  {
+    name      : 'Event Name',
+    valuePath : 'eventName'
+  },
+  {
+    name      : 'Date Issued',
+    valuePath : 'dateIssued'
+  },
+  {
+    name      : 'Amount',
+    valuePath : 'amount'
+  },
+  {
+    name      : 'Date Paid',
+    valuePath : 'datePaid'
+  },
+  {
+    name      : 'View Invoice',
+    valuePath : 'invoiceLink'
+  }
+
+];
+
+dueInvoiceColumns = [
+  {
+    name      : 'Invoice ID',
+    valuePath : 'invoiceId'
+  },
+  {
+    name      : 'Event Name',
+    valuePath : 'eventName'
+  },
+  {
+    name      : 'Date Issued',
+    valuePath : 'dateIssued'
+  },
+  {
+    name      : 'Amount Due',
+    valuePath : 'amount'
+  },
+  {
+    name      : 'View Invoice',
+    valuePath : 'invoiceLink'
+  }
+
+];
+allInvoiceColumns =  [
+  {
+    name      : 'Invoice ID',
+    valuePath : 'invoiceId'
+  },
+  {
+    name      : 'Event Name',
+    valuePath : 'eventName'
+  },
+  {
+    name      : 'Amount',
+    valuePath : 'amount'
+  },
+  {
+    name      : 'Status',
+    valuePath : 'status'
+  }
+
+];
+
+get dueInvoiceRows() {
+  const rows = [];
+  this.model.data.forEach(row => {
```

Comment:
I think we can use a `this.model.data.map` here and for other such iterations too.